backend/src/govsg/services/govsg-callback.service.ts

```diff
@@ -316,8 +316,7 @@ const parseUserMessageWebhook = async (
   body: UserMessageWebhook,
   clientId: WhatsAppApiClient
 ): Promise<void> => {
-  const { wa_id: whatsappId } = body.contacts[0]
-  const { id: messageId, type } = body.messages[0]
+  const { id: messageId, from: whatsappId, type } = body.messages[0]
```

Comment:
Thanks for adding this change. Leaving a [link to the docs](https://developers.facebook.com/docs/whatsapp/on-premises/webhooks/components#messages_object) here for future reference.